Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -156,6 +177,7 @@ function vtkMouseRangeManipulator(publicAPI, model) {
   //-------------------------------------------------------------------------
   publicAPI.onButtonDown = (interactor, renderer, position) => {
     model.previousPosition = position;
+    model.interactionNetDelta = 0;
```

Comment:
Do we also need to initialize `model.interactionNetDelta` in `publicAPI.onScroll` event?